acceptance/redis32/main.go

```diff
@@ -22,16 +23,44 @@ func writeError(w http.ResponseWriter, err error) {
 	message, _ := json.Marshal(map[string]string{
 		"error": err.Error(),
 	})
-	w.Write(message)
 	w.Header().Set("Content-Type", "application/json")
 	w.WriteHeader(http.StatusInternalServerError)
+	w.Write(message)
+}
+
+func newPool(addr string, password string) *redis.Pool {
```

Comment:
Still not sure what the pool is for. Does anything break if we use plain `redis.Dial`?